xarray/tests/test_dataset.py

```diff
@@ -5806,10 +5810,11 @@ def test_reduce_non_numeric(self) -> None:
             "var4" not in data1.mean()
             and "var5" not in data1.mean()
             and "var6" not in data1.mean()
+            and "var7" not in data1.mean()
```

Comment:
Shouldn't `var5` the int64[arrow] one work?

Reply:
Yes that could work. https://docs.xarray.dev/en/stable/internals/duck-arrays-integration.html is another place to update.

For now, please add a comment to the test. I guess we will be back to casting these to numpy quite soon, so we'll end up updating the tests then.